hs_core/templates/pages/my-resources.html

```diff
@@ -10,7 +10,7 @@
             <h2 class="page-title">My Resources</h2>
         </div>
         <div class="col-lg-2" id="facets">
-            <a class="btn btn-success" href="/hsapi/_internal/create-resource"><i class="fa fa-plus"></i> Create new</a>
+            <a class="btn btn-success" href="/create-resource"><i class="fa fa-plus"></i> Create new</a>
```

Comment:
@pkdash Good catch. I don't remember why the url path is changed here, and I just tried to restore the original url path and it still works, so I will restore to the original url path without changing it.